ui/src/app/base/components/VLANSelect/VLANSelect.tsx

```diff
@@ -15,26 +18,44 @@ type Props = {
   defaultOption?: { label: string; value: string } | null;
   fabric?: VLAN["fabric"];
   includeDefaultVlan?: boolean;
+  showSpinnerOnLoad?: boolean;
```

Comment:
adding this prop for compatibility where this pattern is being used (tried to refactor, but this started taking too long)